cmd/exchange_wrapper_issues/main.go

```diff
@@ -0,0 +1,821 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
+	"text/template"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/engine"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+)
+
+var orderTypeOverride string
+var output string
+var orderSideOverride string
+var currencyPairOverride string
+var assetTypeOverride string
+var orderPriceOverride float64
+var orderAmountOverride float64
+var withdrawAddressOverride string
+var authenticatedOnly bool
+var exchangesToUse []string
+
+func parseCLFlags() {
+	exchangesFlag := flag.String("exchanges", "", "A + delimited list of exchange names to run tests against eg -exchanges=bitfinex+anx")
+	assetTypeFlag := flag.String("asset", "", "The asset type to run tests against (where applicable)")
```

Comment:
this could be changed to flag.StringVar(&assetTypeOverride ... ) and some more here if you arnt comparing you can also do this.

Reply:
That's way better 🎉

Reply:
See this as an example: https://github.com/thrasher-corp/gocryptotrader/blob/engine/main.go#L35